### PR TITLE
extra ;

### PR DESCRIPTION
**Pull request by bkentel**

### CODE REVIEW

File: src/debug.cpp [hunk 1]
```diff
@@ -280,8 +280,8 @@ time_info get_time() noexcept {
     timeval tv;
     gettimeofday( &tv, nullptr );
 
-    auto const tt      = tv.tv_sec;
-    auto const current = localtime( &tt );
+    time_t const tt      = tv.tv_sec;
+    auto   const current = localtime( &tt );
```
Comment (KA101) [inline src/debug.cpp]: Separate PR please.  (Moved here from the commit-view)

Comment (narc0tiq) [inline src/debug.cpp]: In fact, you could omit this altogether, or turn the first line into `auto const tt = time_t { tv.tv_sec };` if you're absolutely sure you need the conversion (if so, explain why).

Comment (bkentel) [inline src/debug.cpp]:
```
x86_64-w64-mingw32-g++.exe ...

\src\debug.cpp: In function 'time_info get_time()':
\src\debug.cpp:284:41: error: cannot convert 'const long int*' to 'const time_t* {aka const long long int*}' for argument '1' to 'tm* localtime(const time_t*)'
     auto const current = localtime( &tt );
```
So on a x64 platform the types don't match. Hence the explicit widen to a int64_t type in my little change here. And yes, your suggestion would be perfectly good; better even than this.

Comment (narc0tiq) [inline src/debug.cpp]: I clearly need to add Windows64 as a platform.

**Comment by bkentel** [comment]
Updated to remove the non semi-colon fix.

**Comment by KA101** [comment]
Thanks.

**Pull request merged**